.gitignore

```diff
@@ -40,3 +39,4 @@ build/
 ### Mac OS ###
 .DS_Store
 Thumbs.db
+cp.txt
```

Comment:
there's a way to exclude your own notes with git exclude, thus it will not appear in gitignore and yet be ignored.

<img width="745" height="699" alt="Image" src="https://github.com/user-attachments/assets/4b047c09-fca7-4764-9278-916987e7adf2" />